instrumentation/apache-log4j-2/src/main/java/com/nr/agent/instrumentation/log4j2/AgentUtil.java

```diff
@@ -0,0 +1,136 @@
+/*
+ *
+ *  * Copyright 2022 New Relic Corporation. All rights reserved.
+ *  * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package com.nr.agent.instrumentation.log4j2;
+
+import com.newrelic.agent.bridge.AgentBridge;
+import com.newrelic.api.agent.NewRelic;
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.core.LogEvent;
+import org.apache.logging.log4j.message.Message;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class AgentUtil {
+    public static final int DEFAULT_NUM_OF_LOG_EVENT_ATTRIBUTES = 3;
```

Comment:
Are these constants going to be the same across most logging instrumentation?  Does it make any sense to put in core rather than instrumentation modules?

Reply:
I'm not suggesting doing it now even if it makes sense.  Once we add more instrumentation, perhaps it will make sense to centralize.

Reply:
Yea, they'll be the same for all log libraries. There's no straightforward way to access agent core code in instrumentation modules other than via APIs (either public NewRelic or private AgentBridge). It doesn't make much sense to me to have an API that just exposes a constant like this.